### PR TITLE
Feature/play frame input

### PR DESCRIPTION
**Pull request by tylergibson**
Building from #224 , this completes work to enable both video and image sets to use the Play/Pause/Seek functionality. Also makes the frame count editable so the user can skip to a specific frame in either video or image set.

### CODE REVIEW

File: src/public/js/video-tagging/video-tagging.html [hunk 1]
```diff
@@ -986,6 +1023,41 @@
...
+            if (this.imagelist){//image handling
+                if(this.imageIndex == id - 1) {
+                    //Only fire before step if seeking to the next frame
+                    if(fireEvents)$('#video-tagging').trigger('stepFwdClicked-BeforeStep');    
+                }
+                if(this.imageIndex == id + 1) {
+                    //Only fire before step if seeking to the previous frame
+                    if(fireEvents)$('#video-tagging').trigger('stepBwdClicked-BeforeStep');    
+                }
+                if (id < this.imagelist.length-1) {
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: What happens if id is negative? Do we need to check if id >=0?

File: src/public/js/video-tagging/video-tagging.html [hunk 2]
```diff
@@ -994,7 +1066,7 @@
                 if (this.imageIndex < this.imagelist.length-1) {
                     this.imageIndex++;
                     this.changeImage(`url(file://${this.imagelist[this.imageIndex].replace(/\\/g,"/")})`);
-                    this.playingCallback();
+                    //this.playingCallback();
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: Can probably just remove this

File: src/public/js/video-tagging/video-tagging.html [hunk 3]
```diff
@@ -1019,7 +1091,7 @@
             if (this.imageIndex > 0) {
                 this.imageIndex--;
                 this.changeImage(`url(file://${this.imagelist[this.imageIndex].replace(/\\/g,"/")})`);
-                this.playingCallback();
+                //this.playingCallback();
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: Can probably just remove this

File: src/public/js/video-tagging/video-tagging.html [hunk 4]
```diff
@@ -1054,47 +1126,68 @@
             }
     },
     playState: function() {
-        this.video.play();
-        this.playButton.classList.toggle("icon-pause2", !this.video.paused);
+        console.log("Playing");
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: Can remove debug log

File: src/public/js/video-tagging/video-tagging.html [hunk 5]
```diff
@@ -1054,47 +1126,68 @@
...
         this.optionalTags.toggleEnableButtons(false);
         // $('canvas#overlay').imgAreaSelect({disable: true});//disable canvas
         var self = this;
-        this.playing = setInterval(function() {
-            self.playingCallback();
-        }, 10);
+        
     },
     pauseState: function() {
+        console.log("Paused");
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: Can remove debug log

File: src/public/js/video-tagging/video-tagging.html [hunk 6]
```diff
@@ -1054,47 +1126,68 @@
...
                     this.updateSeekBar();
                 }
-
                 // Update canvas image based on new video frame
                 this.updateFrameCanvas(this.video);
 
             } else{
-                this.frameText.innerHTML = `${this.getCurrentFrameId() + 1}/${this.imagelist.length}`;
-                
+                this.frameText.value = `${this.getCurrentFrameId() + 1}`; //${this.imagelist.length}`;
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: Can remove comment

File: src/public/js/video-tagging/video-tagging.html [hunk 7]
```diff
@@ -1054,47 +1126,68 @@
...
                 this.updateFrameCanvas(this.curImg);
-            }
-                        
+            }           
             this.showAllRegions();
     },
+    playImageSquence: function() {
+        if(this.sequencePlaying) {
+            this.stepFwdClicked();
+            //this.playingCallback();
```
Comment (PIC123) [inline src/public/js/video-tagging/video-tagging.html]: Can remove comment

File: src/public/js/video-tagging/css/sliders.css [hunk 8]
```diff
@@ -142,13 +142,13 @@ input[type='range']::-ms-tooltip {
 
 
 /*webkit and moz - show seek bar button on hover or click*/
-.seek::-webkit-slider-thumb {
+/* .seek::-webkit-slider-thumb {
```
Comment (PIC123) [inline src/public/js/video-tagging/css/sliders.css]: Can we remove this then?

Comment (tylergibson) [reply src/public/js/video-tagging/css/sliders.css]: Yep, done

File: src/public/js/video-tagging/video-tagging.html (hunk 1 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: id must be >= 0, it's tied to the imagelist index which is zero bound

File: src/public/js/video-tagging/video-tagging.html (hunk 2 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: Good catch, done

File: src/public/js/video-tagging/video-tagging.html (hunk 3 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: Good catch, done

File: src/public/js/video-tagging/video-tagging.html (hunk 4 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: Done

File: src/public/js/video-tagging/video-tagging.html (hunk 5 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: Got it

File: src/public/js/video-tagging/video-tagging.html (hunk 6 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: Got it

File: src/public/js/video-tagging/video-tagging.html (hunk 7 above)
Comment (tylergibson) [reply src/public/js/video-tagging/video-tagging.html]: Got it

**Pull request merged**